edx_sga/sga.py

```diff
@@ -57,92 +61,103 @@ def reify(meth):
     Keyword arguments:
     inst
     """
+
     def getter(inst):
         """
         Set value to meth name in dict and returns value.
         """
         value = meth(inst)
         inst.__dict__[meth.__name__] = value
         return value
+
     return property(getter)
 
 
-class StaffGradedAssignmentXBlock(StudioEditableXBlockMixin, ShowAnswerXBlockMixin, XBlock):
+class StaffGradedAssignmentXBlock(
+    StudioEditableXBlockMixin, ShowAnswerXBlockMixin, XBlock
+):
     """
     This block defines a Staff Graded Assignment.  Students are shown a rubric
     and invited to upload a file which is then graded by staff.
     """
+
     has_score = True
-    icon_class = 'problem'
+    icon_class = "problem"
     STUDENT_FILEUPLOAD_MAX_SIZE = 4 * 1000 * 1000  # 4 MB
-    editable_fields = ('display_name', 'points', 'weight', 'showanswer', 'solution')
+    editable_fields = ("display_name", "points", "weight", "showanswer", "solution")
 
     display_name = String(
         display_name=_("Display Name"),
-        default=_('Staff Graded Assignment'),
+        default=_("Staff Graded Assignment"),
         scope=Scope.settings,
-        help=_("This name appears in the horizontal navigation at the top of "
-               "the page.")
+        help=_(
+            "This name appears in the horizontal navigation at the top of " "the page."
+        ),
     )
 
     weight = Float(
         display_name=_("Problem Weight"),
-        help=_("Defines the number of points each problem is worth. "
-               "If the value is not set, the problem is worth the sum of the "
-               "option point values."),
-        values={"min": 0, "step": .1},
-        scope=Scope.settings
+        help=_(
+            "Defines the number of points each problem is worth. "
+            "If the value is not set, the problem is worth the sum of the "
+            "option point values."
+        ),
+        values={"min": 0, "step": 0.1},
+        scope=Scope.settings,
     )
 
     points = Integer(
         display_name=_("Maximum score"),
         help=_("Maximum grade score given to assignment by staff."),
         default=100,
-        scope=Scope.settings
+        scope=Scope.settings,
     )
 
     staff_score = Integer(
         display_name=_("Score assigned by non-instructor staff"),
-        help=_("Score will need to be approved by instructor before being "
-               "published."),
+        help=_(
+            "Score will need to be approved by instructor before being " "published."
+        ),
         default=None,
-        scope=Scope.settings
+        scope=Scope.settings,
     )
 
     comment = String(
         display_name=_("Instructor comment"),
-        default='',
+        default="",
         scope=Scope.user_state,
-        help=_("Feedback given to student by instructor.")
+        help=_("Feedback given to student by instructor."),
```

Comment:
same here

Reply:
I didn't manually add that. It was introduced by running `black`